### DISCUSSION TITLE
Documentation is misleading

### DISCUSSION THREAD

**Discussion by chandru89new** [vercel/vercel#8807] [General] [answered]
https://vercel.com/docs/rest-api#endpoints/teams/get-access-request-status

The `{userId}` part is actually required but marked as `optional` in the documentation.

I raised an issue about this almost a month ago, got a reply from Vercel's support team saying they will fix it and so far, nothing has happened.

**Comment by amyegan** [comment, marked as answer]
Thanks for pointing this out @chandru89new! I'm opening a new internal issue with these details and the ones from https://github.com/vercel/vercel/discussions/8806. I can't find the older issue you raised referenced, but if you share it here I will link back to it from the new ticket.

**Reply by chandru89new** [reply]
Thanks for your reply, @amyegan.

Wrt to the older issue, I had raised it via -> Issues -> New Bug report (which takes you to a Vercel form). I got a reply saying they will look into it but nothing's changed since.

**Reply by amyegan** [reply]
Hi again @chandru89new! I got some more information from the documentation and build teams. The `userId` really is supposed to be optional. Can you share the error message you are receiving? 

We do expect errors if you are not authenticated or recognized as part of the team, but other unexpected errors are possible. Knowing what error you see will help the team determine what is going wrong in your case so they can come up with a solution (or, at least, a more helpful error message!)

**Reply by chandru89new** [reply]
Hi @amyegan, if the userId is optional, how would you build the endpoint? userId is very part of the endpoint itself and not some query param so it becomes impossible to make a request to this endpoint without the userId.

**Reply by amyegan** [reply]
If a `userId` path segment is not provided, the endpoint returns the status of the authenticated user. The [get access request status](https://vercel.com/docs/rest-api#endpoints/teams/get-access-request-status) endpoint documentation has just been updated with this information as well. Thanks for bringing this up to improve our docs!

**Reply by chandru89new** [reply]
Ah okay that makes sense now. Thanks for letting me know + updating the docs, @amyegan!
